plugin/dnstap/handler.go

```diff
@@ -16,11 +16,14 @@ import (
 // Dnstap is the dnstap handler.
 type Dnstap struct {
 	Next plugin.Handler
-	Out  io.Writer
+	IO   IOThread
```

Comment:
hmmm, don't call it Thread, we don't have those in Go (well the language, the underlying runtime obv has them, but we don't see them). Don't have an immediate idea for a better name.

Reply:
Maybe just call the interface Sender and rename the Dnstap method to Send? So, it will reflect the purpose of the interface, not implementation details